ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/snmp.yml

```diff
@@ -62,25 +62,28 @@ snmp_server:
       priv: aes
       priv_passphrase: 7a07246a6e3467909098d01619e076ad
   hosts:
+    - host: 10.6.75.121
+      vrf: MGMT
+      version: 1
+      communities:
+        community: SNMP-COMMUNITY-1
+    - host: 10.6.75.121
+      vrf: MGMT
+      version: 2c
+      communities:
+        community: SNMP-COMMUNITY-2
     - host: 10.6.75.99
-      vrf: mgt
+      vrf: MGMT
+      version: 3
```

Comment:
I suggest making `version` field default to `3` so existing deployments would not break. So you could add a fourth host without the version field set.

Reply:
Maybe not a concern, but the EOS default when no version is set is 2c.  I think setting to 3 makes sense to prevent breakage of existing deployments, but could lead to invalid EOS CLI.  I re-wrote the j2 template so that if a username is set, the default is v3, and if a community is set, it defaults to 2c.  This may be too much logic, but it seems to be the best way to me to generate valid eos config.